Modules/Filtering/ImageGrid/include/itkExpandImageFilter.h

```diff
@@ -50,15 +50,11 @@ namespace itk
  * This filter is implemented as a multithreaded filter and supports
  * streaming.
  *
- * \warning This filter only works for image with scalar pixel types.
- * For vector images use VectorExpandImageFilter.
- *
```

Comment:
Do we have a test with Image of Vector's or itk::VectorImage some place? 

I just double checked and SimpleITK is running this filter separately for each component instead of instantiating it with a `VectorImage`.

Reply:
From the diff, it looks like it is used by `itkFEMRegistrationFilter.h`, but I created #1066 to track this.